.github/workflows/deploy-github-page.yml

```diff
@@ -76,6 +78,8 @@ jobs:
           micromamba activate xeus-lite-host
           python -m pip install jupyterlite-xeus
           jupyter lite build --XeusAddon.prefix=${{ env.PREFIX }} --output-dir dist
+          #cp xcpp.data dist/extensions/@jupyterlite/xeus/static
+          #cp $PREFIX/lib/libclangCppInterOp.so dist/extensions/@jupyterlite/xeus/static
```

Comment:
We need these changes but I am not sure as to how the deploy pages works ?
What is responsible for executing the `jupyter lite serve` command. 

Check the changes made in README.md to understand that these are essential changes and need to go in before serving.